src/containerapp/azext_containerapp/_validators.py

```diff
@@ -41,9 +41,13 @@ def _is_number(s):
 
 def validate_revision_suffix(value):
     if value is not None:
-        matched = re.match(r"^[a-z](?!.*-{2})([-a-z0-9]*[a-z0-9])?$", value)
+        # what does the following regex check?
+        # 1. ^[a-z0-9] - starts with a letter or number
+        # 2. (?!.*-{2}) - does not contain '--'
+        # 3. ([-a-z0-9]*[a-z0-9])? - ends with a letter or number and can contain '-' in between
+        matched = re.match(r"^[a-z0-9](?!.*-{2})([-a-z0-9]*[a-z0-9])?$", value)
         if not matched:
```

Comment:
Could you please add some test about this validate?

Reply:
Added